frontend/src/app/libs/landing/services/event-converter.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@angular/core';
+import { Event } from '../models/event';
+import { EventDto } from '../models/eventDto';
+
+@Injectable({
+	providedIn: 'root',
+})
+export class EventConverter {
+	public convertFromDto(eventsDto: EventDto[]): Event[] {
+		return eventsDto.map((dto: EventDto) => {
+			return (({ _id, tasks, participations, maxScore, isActive, users, ...event }: EventDto) => ({ id: _id, ...event }))(dto);
```

Comment:
why not just
`.map(({_id,  ...dto) => { 
return { id: _id, ...dto };
})` ?

Reply:
because we not only need to rename the _id key, but also remove unnecessary fields( tasks, participations, maxScore, isActive, users)